contrib/epee/include/wipeable_string.h

```diff
@@ -79,6 +86,20 @@ namespace epee
   private:
     std::vector<char> buffer;
   };
+
+  template<typename T> inline bool wipeable_string::hex_to_pod(T &pod) const
```

Comment:
I don't understand why this was created - can the existing functions in epee be used for this purpose?

Reply:
Its workhorse is different. I suppose it could be selected by some template function somewhere though.